src/analyze/analyze.c

```diff
@@ -559,6 +560,10 @@ static int parse_argv(int argc, char *argv[]) {
                 return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
                                        "Option --offline= is only supported for security right now.");
 
+        if (arg_offline && optind >= argc - 1)
```

Comment:
Maybe better to merge the previous condition?
And better to use strv_isempty() with strv_skip() or something similar?

Reply:
> Maybe better to merge the previous condition?

Hmm, I'm not sure about this, do you want to merge two different error messages into one like "Option --offline= is only supported for security right now, and requires one or more units to perform a security review."?

> And better to use strv_isempty() with strv_skip() or something similar?

I simply followed the way that it's used below in the same file:

https://github.com/systemd/systemd/blob/e992753e581cf6b94cd69122085abc54e26b197d/src/analyze/analyze.c#L595-L599

Is `strv_isempty(strv_skip(...))` a change that should be made tree wide?

Reply:
OK, then feel free to keep the block as is.